spec/models/item_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Item, type: :model do
+  let(:user) { FactoryBot.create :user }
+
+  describe '#valid?' do
+    subject { item.valid? }
+
+    context 'パラメータが全て正当であるとき' do
+      let(:item) { described_class.new name:, description:, file:, user: }
+      let(:name) { 'foo.txt' }
+      let(:description) { Faker::Lorem.characters }
+      let(:file) { fixture_file_upload 'empty.txt' }
+
+      it { is_expected.to be_truthy }
+    end
+
+    context 'ファイルが指定されていないとき' do
+      let(:item) { described_class.new file:, user: }
+      let(:file) { nil }
+
+      it { is_expected.to be_falsey }
+    end
+
+    context '同名のファイルが存在するとき' do
+      let(:item) { described_class.new file:, user: }
+      let(:file) { fixture_file_upload 'empty.txt' }
+
+      before do
+        described_class.new(file:, user:).save
```

Comment:
こう書くのが良さそう?
```suggestion
        described_class.create!(file:, user:)
```

* new & save より create の方が一度で済むのでシンプルでは
* create より create! の方がエラーに気づきやすいのでよいのでは (もしくは save より save! の方が〜)
    * create も save もバリデーションエラーなどの際は false を返すだけなのでサイレントに失敗します

Reply:
確かにそうですね。修正します。